### PR TITLE
feat: implement add custom sudo key for parachain

### PR DESCRIPTION
**Pull request by abhiyana**
### Commit Message

```bash
feat: implement add custom sudo key for parachain
```

see the [guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages) for commit messages.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have documented my code in accordance with the [documentation guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#documentation)
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the unit tests
- [ ] I only have one commit (if not, squash them into one commit).
- [x] I have a descriptive commit message that adheres to the [commit message guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages)

> Please review the [CONTRIBUTING.md](/CONTRIBUTING.md) file for detailed contributing guidelines.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -11,46 +11,43 @@
...
+          "key": {
+            "private_phrase":"diamond crane pioneer aisle conduct media news cup price olive dust light" ,
+            "public_key":"5GgwsM48CCT7oPUyFTTpE5L1t7Fjpe7D8SDNKxJxug15YEKy"
+          }
         }
-      ]
+      ],
+      "sudo_key": {
+        "private_phrase":"dizzy rose offer wall social glory debris gift govern seminar almost bicycle" ,
+        "public_key":"5G3vA9UC6J7hPN5Jqfiz4qFNqYAfahHxffbuQf6dcZZx3Yx6"
```
Comment (shanithkk) [inline config.json:46]: public key can be generated using the seed

File: parachain/static_files/javascript/edit_parachain_plain.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+const fs = require("fs");
+
+function updateParachainSpec(PARA_SPEC_FILE, PARA_ID, NEW_SUDO_KEY, INITIAL_COLLATORS) {
```
Comment (shanithkk) [inline parachain/static_files/javascript/edit_parachain_plain.js:3]: follow naming conventions

File: parachain/static_files/javascript/edit_parachain_plain.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  try {
+    const rawdata = fs.readFileSync(PARA_SPEC_FILE);
```
Comment (shanithkk) [inline parachain/static_files/javascript/edit_parachain_plain.js:5]: naming convention

**Review by shanithkk — CHANGES_REQUESTED** [review]

File: config.json (hunk 1 above)
Comment (abhiyana) [reply config.json:46]: Change implemented

File: parachain/static_files/javascript/edit_parachain_plain.js (hunk 2 above)
Comment (abhiyana) [reply parachain/static_files/javascript/edit_parachain_plain.js]: updated

File: config.json [hunk 4]
```diff
@@ -11,46 +11,43 @@
...
-          "prometheus": true
+          "prometheus": false,
+          "key": {
+            "private_phrase":"diamond crane pioneer aisle conduct media news cup price olive dust light" ,
+            "public_key":"5GgwsM48CCT7oPUyFTTpE5L1t7Fjpe7D8SDNKxJxug15YEKy"
+          }
         }
-      ]
+      ],
+      "sudo_key": {
```
Comment (shanithkk) [inline config.json:44]: without sudo key throwing error, can you make the sudo key as optional

**Review by shanithkk — CHANGES_REQUESTED** [review]

**Review by shanithkk — APPROVED** [review]

**Review by shreyasbhat0 — APPROVED** [review]

**Pull request merged**